assets/foliate-js/src/book.js

```diff
@@ -1167,7 +1167,22 @@ class Reader {
   }
 
   #onTouchEnd = ({ detail: e }) => {
-    if (this.#ignoreTouch()) return;
+    if (this.#ignoreTouch()) {
+      if (e.touchState.direction === 'vertical') {
+        const renderer = this.view.renderer;
+        const scrollTop = renderer.shadowRoot.querySelector('#container').scrollTop;
+        const deltaY = e.touchState.delta.y;
+        const swipeThreshold = 60;
+
+        if (deltaY > swipeThreshold && scrollTop <= 1) {
+          renderer.shadowRoot.querySelector('#container').scrollTop = 0;
```

Comment:
`view.prevPage()` (via `Paginator.scrollPrev`) checks whether `renderer.start > 0` to decide whether to scroll up or change the page.  
The iOS rubber banding makes `renderer.start` positive (150 > 0), which forces an unnecessary scroll instead of changing the page.  
Forcing `scrollTop = 0` before the call ensures that the condition is met to change the page.